packages/contracts/helpers/contracts-deployments.ts

```diff
@@ -1031,7 +1031,7 @@ export const deployMockAggregator = async (
 export const deployVMEXOracle = async (verify?: boolean) =>
   withSaveAndVerify(
     await new VMEXOracleFactory(await getFirstSigner()).deploy(),
-    eContractid.VMEXOracle,
+    eContractid.VMEXOracleImpl,
```

Comment:
Does this break any existing code? Seems like places will use `eContractid.VMEXOracle`

The `eContractid.VMEXOracle` is no longer being used, so every usage of it should be changed?

Reply:
it makes it more clear that the stored address is for the implementation. This doesn't break any code. deployVMEXOracle is meant to deploy the impl, and originally aave had it just named confusingly

Reply:
Ah ok, you should change the name of the function to be more clear then.

It doesn't matter if aave had it named this way. If our way is cleaner let's make it cleaner